front-end/src/main/kotlin/org/jetbrains/kotlin/spec/entity/test/Test.kt

```diff
@@ -0,0 +1,14 @@
+package org.jetbrains.kotlin.spec.entity.test
+
+import org.jetbrains.kotlin.spec.entity.test.parameters.TestInfo
+import org.jetbrains.kotlin.spec.entity.test.parameters.TestType
+import org.jetbrains.kotlin.spec.entity.test.parameters.testArea.TestArea
+
+class Test(val testInfo: TestInfo, val testPlace: TestPlace, content: String, testArea: TestArea) {
```

Comment:
I think `Test` is also not so good name due to too much abstractness. Use `SpecTest`.

Reply:
done